ArticleTemplates/assets/js/shim.js

```diff
@@ -1,92 +1,97 @@
+        'articleTagInserter',
+        'articleOutbrainInserter',
+        'audioBackground',
+        'superAudioSlider',
+        'audioPlay',
+        'audioStop',
+        'commentsFailed',
+        'articleCommentsFailed',
+        'commentsClosed',
+        'commentsOpen',
+        'showMoreTags',
+        'commentsReplyFormatting',
+        'footballGoal',
+        'footballStatus',
+        'footballMatchInfo',
+        'footballMatchInfoFailed',
+        'liveblogDeleteBlock',
+        'liveblogNewBlock',
+        'getMpuPosCallback',
+        'initMpuPoller',
+        'videoPositioning',
+        'getArticleHeight'
+    ];
 
-var nativeFunctionCalls = [
-    'articleCommentsInserter',
-    'articleCardsInserter',
-    'articleTagInserter',
-    'articleOutbrainInserter',
-    'audioBackground',
-    'superAudioSlider',
-    'audioPlay',
-    'audioStop',
-    'commentsFailed',
-    'articleCommentsFailed',
-    'commentsClosed',
-    'commentsOpen',
-    'showMoreTags',
-    'commentsReplyFormatting',
-    'footballGoal',
-    'footballStatus',
-    'footballMatchInfo',
-    'footballMatchInfoFailed',
-    'liveblogDeleteBlock',
-    'liveblogNewBlock',
-    'getMpuPosCallback',
-    'initMpuPoller',
-    'videoPositioning',
-    'getArticleHeight'
-];
+    Array.prototype.forEach.call(nativeFunctionCalls, function (name) {
+        // Create a function to catch early calls
+        window[name] = function () {
+            // Create or get the queue for this function
+            var queue = name + 'Queue';
+            window[queue] = window[queue] || [];
+            // Store arguments for each call so
+            // true function can apply these when ready
+            window[queue].push(arguments);
+        };
+    });
 
-Array.prototype.forEach.call(nativeFunctionCalls, function (name) {
-    // Create a function to catch early calls
-    window[name] = function () {
-        // Create or get the queue for this function
-        var queue = name + 'Queue';
-        window[queue] = window[queue] || [];
-        // Store arguments for each call so
-        // true function can apply these when ready
-        window[queue].push(arguments);
+    window.applyNativeFunctionCall = function (name) {
+        var queue = window[name + 'Queue'];
+        
+        if (queue) {
+            Array.prototype.forEach.call(queue, function (item) {
+                window[name].apply(this, item);
+            });
+        }
     };
-});
 
-var applyNativeFunctionCall = function (name) {
-    var queue = window[name + 'Queue'];
-    if (queue) {
-        Array.prototype.forEach.call(queue, function (item) {
-            window[name].apply(this, item);
-        });
-    }
-};
-
-window.animFrame = window.requestAnimationFrame || function( callback ){
-    window.setTimeout(callback, 1000 / 60);
-};
+    window.animFrame = window.requestAnimationFrame || function( callback ){
+        window.setTimeout(callback, 1000 / 60);
```

Comment:
I think it was originally not intended as a genuine shim. It was only used once, to kickoff the initial bootstrap. However it looks like this has now been used elsewhere in a context where a genuine requestAnimationFrame is required so I'll look at including the Paul Irish shim in another branch I'm working on now which touches this code.